docs/content/5.integrations/2.guide/2.adding-headers-to-request.md

```diff
@@ -0,0 +1,284 @@
+# Pass Custom Headers in Requests
+
+In this tutorial, we'll demonstrate how to set up an interceptor to pass headers when making requests using the axios client inside the Vue Storefront SDK. This is useful when you need to append custom headers or cookies to your API calls.
+
+After passing the headers to the axios client, we'll also show you how to pass them to external services using the API Client.
+
+::danger
+Do not use createSharedComposable if creating a shared composable that uses the interceptor. This will cause the interceptor to only be created once, which will result in the headers being shared across instances.
+::
+
+## Prerequisites:
+Ensure you have the Vue Storefront SDK set up in your project.
+
+## Notes
+::info
+This guide uses the integrations boilerplate with the nuxt option. The same basic principles apply to other frameworks, but the implementation may vary.
```

Comment:
```suggestion
This guide uses the [Nuxt Storefront Boilerplate](https://github.com/vuestorefront/storefront-nuxt3-boilerplate). for demonstration purposes. The same basic principles apply to other frameworks using the Vue Storefront stack, but the implementation may slightly vary.
```

Reply:
The guide does not use the nuxt 3 boilerplate. It uses the integrations boilerplate with the nuxt option.